test/integration/helpers/Paths.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Appium.Net.Integration.Tests.helpers;
+using Appium.Net.Integration.Tests.Properties;
+using System.IO;
+using System.Text;
+
+namespace Appium.Net.Integration.Tests.Helpers
+{
+    internal class Paths
+    {
+        private string _pathToAppiumPackageIndex;
+
+        public string PathToAppiumPackageIndex
+        {
+            get
+            {
+                if (_pathToAppiumPackageIndex == null)
+                {
+                    GetAppiumJsPath();
+                }
+                return _pathToAppiumPackageIndex;
+            }
+        }
+
+        private void GetAppiumJsPath()
+        {
+            byte[] bytes;
```

Comment:
not sure about conventions in C#, but I usually define vars close to places where they are actually used